tests/small/LandIce/FO_Hydrology/input_slab_steadyT.yaml

```diff
@@ -181,6 +180,10 @@ ANONYMOUS:
             Field Name: sliding_velocity
             Field Type: Node Scalar
             Field Usage: Output
+          Field 10:
+            Field Name: water_pressure
+            Field Type: Node Scalar
+            Field Usage: Output
   Piro:
```

Comment:
No Regression sublist in this test, can you add it?

Reply:
Ah, this input file got committed, but there's not actual test added. Since I haven't really checked this input file "works", I might simply remove it.